Dashboard/citymanage/static/js/dashboard.js

```diff
@@ -0,0 +1,69 @@
+window.onload = function initMap() {
+
+    var stations_locations = new Array([]);
```

Comment:
Camel naming should be used instead of underscore according to our JavaScript coding standard